server/main/templates/main/study-overview.html

```diff
@@ -154,42 +152,25 @@
           Other line creation options:
           {% endblocktrans %}
         </p>
-        <a class="dropLink" href="{% url 'main:describe:describe' slug=study.slug %}">
-          <button
-            class="btn btn-default combosButton"
-            id="combosButton"
-            name="action"
-            type="button"
-          >
-            <span class="fas fa-random"></span>
-            {% trans "Add Line Combo's" %}
-          </button>
+        <a class="dropLink btn btn-default combosButton" href="{% url 'main:describe:describe' slug=study.slug %}"
+          id="combosButton"
+        >
+          <span class="fas fa-random"></span>
+          {% trans "Add Line Combo's" %}
         </a>
       </div>
       <div>
-        <a class="dropLink" href="{% url 'main:lines' slug=study.slug %}">
-          <button
-            class="btn btn-default"
-            id="manualButton"
-            name="action"
-            type="button"
-          >
-            <span class="fas fa-pencil-alt"></span>
-            {% trans 'Enter Manually' %}
-          </button>
+        <a class="dropLink btn btn-default" href="{% url 'main:lines' slug=study.slug %}"
+          id="manualButton"
+        >
+          <span class="fas fa-pencil-alt"></span>
+          {% trans 'Enter Manually' %}
         </a>
       </div>
       <div>
-        <a class="dropLink" href="#">
-          <button
-            class="btn btn-default dz-browse-link"
-            id="browseButton"
-            name="action"
-            type="button"
-          >
-            <span class="fas fa-folder-open"></span>
-            {% trans 'Browse for file' %}
-          </button>
+        <a class="dropLink btn btn-default dz-browse-link" href="#"
+          id="browseButton"
+        >
```

Comment:
This one looks like it no longer has any content? The others are un-nesting the button into the anchor and keeping their content.